src/jsMain/kotlin/ui/App.kt

```diff
@@ -229,6 +230,7 @@ private fun getSelectedDependencies(deps: Map<Dependency, StateInstance<Boolean>
                     SQLDelightDriverNative,
                     SQLDelightDriverJs
                 )
+                d.group == "com.apollographql.apollo3" -> listOf(ApolloRuntime)
```

Comment:
I think there should be a plugin and runtime library, right?

Reply:
Correct, thanks!